### ISSUE TITLE
Incorrect tee command ?

### ISSUE THREAD

**Issue by chinkung** [sakai135/wsl-vpnkit#7]
It looks like your tee command in document is not working on Ubuntu

![image](https://user-images.githubusercontent.com/2522806/98194763-b63a8580-1f52-11eb-8e76-0c2dce21d682.png)

I think it should be changed to

```
$ sudo tee /etc/wsl.conf > /dev/null <<EOL
[network]
generateResolvConf = false
EOL
```

**Comment by chinkung** [comment]
I have create PR for this https://github.com/sakai135/wsl-vpnkit/pull/8

**Comment by sakai135** [comment]
Yup, thanks for pointing it out. The command I was using was the following, and I never noticed I had it wrong in my readme.

```sh
$ cat <<EOL | sudo tee /etc/resolv.conf
nameserver 192.168.67.1
EOL
```

**Comment by NicoVogel** [comment]
I also noticed the wrong command.

> Yup, thanks for pointing it out. The command I was using was the following, and I never noticed I had it wrong in my readme.
> 
> ```shell
> $ cat <<EOL | sudo tee /etc/resolv.conf
> nameserver 192.168.67.1
> EOL
> ```

This did not work for me (WSL2, Ubuntu 20.04, Windows Terminal).
I used the following approach:

```shell
sudo tee /etc/wsl.conf <<EOL
[network]
generateResolvConf = false
EOL
```

Later I checked the approach of @chinkung, which also worked fine.
My source: [How to append multiple lines to a file](https://unix.stackexchange.com/questions/77277/how-to-append-multiple-lines-to-a-file)

**Comment by sakai135** [comment]
Resolved with #8
